frontend/templates/Chat/DiscoveryLibraryWindow/DiscoveryLibraryWindow.jsx

```diff
@@ -0,0 +1,391 @@
+        messagesContainerRef.current.scrollHeight -
+          messagesContainerRef.current.clientHeight -
+          messagesContainerRef.current.scrollTop
+      ) <= 1;
+
+    if (fullyScrolled !== scrolled) dispatch(setFullyScrolled(scrolled));
+  };
+
+  const handleScrollToBottom = () => {
+    messagesContainerRef.current?.scrollTo(
+      0,
+      messagesContainerRef.current?.scrollHeight,
+      {
+        behavior: 'smooth',
+      }
+    );
+
+    dispatch(setStreamingDone(false));
+  };
+  /* Push Enter */
+  const keyDownHandler = async (e) => {
+    if (typing || !input || streaming) return;
+    if (e.keyCode === 13) handleSendMessage();
+  };
+
+  const renderQuickAction = () => {
+    return (
+      <InputAdornment position="start">
+        <Grid
+          onClick={() => dispatch(setDisplayQuickActions(!displayQuickActions))}
+          {...styles.quickActionButton}
+        >
+          <AddIcon {...styles.quickActionButtonAddIcon} />
+          <Typography>Actions</Typography>
+        </Grid>
+      </InputAdornment>
+    );
+  };
+
+  const renderSendIcon = () => {
+    return (
+      <InputAdornment position="end">
+        <IconButton
+          onClick={handleSendMessage}
+          {...styles.bottomChatContent.iconButtonProps(
+            typing || error || !input || streaming
+          )}
+        >
+          <NavigationIcon />
+        </IconButton>
+      </InputAdornment>
+    );
+  };
+
+  const renderCustomPrompt = () => {
+    if (selectedPrompt) {
+      return (
+        <Grid
+          onClick={() => dispatch(setMore({ role: 'shutdown' }))}
+          {...styles.centerChat.centerChatGridProps}
+        >
+          <Grid {...styles.cardGridProps}>
+            <Grid {...styles.cardContent}>
+              <Typography {...styles.cardTitleProps}>
+                {selectedPrompt.title}
+              </Typography>
+              <Grid item {...styles.unionPurpleIcon}>
+                <UnionPurpleIcon />
+              </Grid>
+            </Grid>
+          </Grid>
+        </Grid>
+      );
+    }
+    return null;
+  };
+  const renderStartChatMessage = () => {
+    return (
+      <TextMessage
+        isMyMessage={false}
+        message="Hi! What would you like to build today?**"
+      />
+    );
+  };
+
+  const renderCenterChatContent = () => {
+    if (selectedPrompt) {
+      return renderCustomPrompt(); // Render custom prompt if selected
+    }
+    return (
+      <Grid
+        onClick={() => dispatch(setMore({ role: 'shutdown' }))}
+        {...styles.centerChat.centerChatGridProps}
+      >
+        <Grid
+          ref={messagesContainerRef}
+          onScroll={handleOnScroll}
+          {...styles.centerChat.messagesGridProps}
+        >
+          {/* Render the start chat message if there are no chat messages or if the info chat is not open. */}
+          {(chatMessages?.length === 0 || !chatMessages) && !infoChatOpened
+            ? renderStartChatMessage()
+            : null}
```

Comment:
You can just write:

`(chatMessages?.length === 0 || !chatMessages) && !infoChatOpened && renderStartChatMessage()`